Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -391,8 +361,21 @@ def bump_formula_pr
             #{user_message}
           EOS
         end
-        safe_system "hub", "pull-request", *hub_args, "-m", pr_message
-        safe_system "git", "checkout", *git_final_checkout_args, "-"
+        pr_title = "#{formula.name} #{new_formula_version}#{devel_message}"
+
+        begin
+          url = GitHub.create_pull_request(formula.tap.full_name, pr_title,
+                                     "#{username}:#{branch}", "master", pr_message)["html_url"]
+
+          if ARGV.include?("--no-browse")
+            puts "Pull Request: #{url}"
```

Comment:
If !$stdout.tty? I think we should just output the url.

Reply:
Agreed.